deeppavlov_agent/http_api/__init__.py

```diff
@@ -21,7 +21,8 @@ def app_factory(pipeline_configs=None, debug=None, response_time_limit=None):
     app = init_app(
         agent=agent, session=session, consumers=workers,
         logger_stats=response_logger, output_formatter=output_formatter,
-        debug=debug or DEBUG, response_time_limit=response_time_limit or TIME_LIMIT
+        debug=debug or DEBUG, response_time_limit=response_time_limit or TIME_LIMIT,
+        cors=cors or CORS
```

Comment:
What if `cors` is `False` but `CORS` is `True`?
Same for `debug` and `response_time_limit ` actually
```suggestion
        debug=debug if debug is not None else DEBUG,
        response_time_limit=response_time_limit if response_time_limit is not None else TIME_LIMIT,
        cors=cors if cors is not None else CORS
```